assets/src/dashboard/components/cardGallery/index.js

```diff
@@ -149,7 +140,7 @@ function CardGallery({ children }) {
 }
 
 CardGallery.propTypes = {
-  children: PropTypes.node.isRequired,
+  template: StoryPropType.isRequired,
```

Comment:
Nit, since this card gallery would work with a story too (and not just a template), maybe we should call this `story` just so we're using the most generic nomenclature for the type.